cl/recap_rss/tasks.py

```diff
@@ -339,7 +340,7 @@ def merge_rss_feed_contents(self, feed_data, court_pk, metadata_only=False):
                 # The item is already in the cache, ergo it's getting processed
                 # in another thread/process and we had a race condition.
                 continue
-            d = find_docket_object(
+            d = async_to_sync(find_docket_object)(
```

Comment:
And why call it here? 

Sorry if these are remedial questions...

Reply:
This is because `merge_rss_feed_contents` is a non-async function while `find_docket_object` is an async function, so we have to adapt here when calling async code from non-async.